src/ia-bookreader/ia-bookreader.js

```diff
@@ -26,45 +32,101 @@ export class BookReader extends LitElement {
     this.bookreader = undefined;
     this.baseHost = 'https://archive.org';
     this.fullscreen = false;
+    this.signedIn = false;
     /** @type {ModalManager} */
     this.modal = undefined;
     /** @type {SharedResizeObserver} */
-    this.sharedObserver = new SharedResizeObserver();
+    this.sharedObserver = undefined;
+    this.loaded = false;
+    this.menuShortcuts = [];
+    this.menuContents = [];
   }
 
-  firstUpdated() {
-    this.createModal();
-  }
+  updated() {
+    if (!this.modal) {
+      this.modal = new ModalManager();
+    }
 
-  /** Creates modal DOM & attaches to `<body>` */
-  createModal() {
-    this.modal = document.createElement(
-      'modal-manager'
-    );
-    document.body.appendChild(this.modal);
+    if (!this.sharedObserver) {
+      this.sharedObserver = new SharedResizeObserver();
+    }
   }
-  /* End Modal management */
+
 
   manageFullscreen(e) {
     const { detail } = e;
     const fullscreen = !!detail.isFullScreen;
     this.fullscreen = fullscreen;
+    this.dispatchEvent(new CustomEvent('fullscreenStatusUpdated', { detail: { fullscreen }}));
```

Comment:
suggestion: remove `...status...` => `fullscreenUpdated` or use `...state...` as in `loadingStateUpdated`

Reply:
it's local to the web component chain, so book-nav, not core br...yet.